test/EndToEnd/tests/ServicesTest.ps1

```diff
@@ -703,6 +703,40 @@ function Test-BatchEventsApi
     Assert-True $result
 }
 
+function Test-ExecuteInitScriptsPerSolution
```

Comment:
I don't think there is any way to actually verify that install.ps1 loads all init.ps1 scripts first. Since the PM console has to be open to run E2E tests it should always load these first. You can't test the background host scenario.